### DISCUSSION TITLE
How to sign in?

### DISCUSSION THREAD

**Discussion by SupersonicMagpie** [community/community#7356] [Copilot Conversations]
Copilot asks me to sign in:
![image](https://user-images.githubusercontent.com/48965391/140499328-88d39d24-47dd-4f07-bfc8-5002154adff9.png)
Then it gives me a prompt, i click "allow" and it asks me to enter the token:
![image](https://user-images.githubusercontent.com/48965391/140499476-2ecada5a-0758-44e8-a1ca-14dd737720e5.png)
However, I can't find where to create get this token from or how can I create it

**Comment by merlijnvanlent** [comment]
Maybe this will help!
https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

**Reply by avigloz** [reply]
Yeah, this is it. 

tl;dr: Top right -> Settings -> Developer Settings -> Personal Access Tokens

I believe copilot needs repo + user permissions.

**Reply by SupersonicMagpie** [reply]
when i did so it gave me this: 
![image](https://user-images.githubusercontent.com/48965391/140714850-fb9b323f-cdb1-454f-a9f4-d17834f76549.png)
and a few days ago i got this:
![image](https://user-images.githubusercontent.com/48965391/140714923-f5d2a1d5-8c95-45a7-8345-3e822ff020bf.png)

So here is my feedback: make an instalation/login process simpler, before inviting the users to test it. Just compare this process and amount of such issues to any other existing VSCode extention. Seems like copilot is the hardest one.
